doc/rados/operations/pools.rst

```diff
@@ -55,21 +55,23 @@ To list your cluster's pools, execute:
 Create a Pool
 =============
 
-Before creating pools, refer to the `Pool, PG and CRUSH Config Reference`_.
-Ideally, you should override the default value for the number of placement
-groups in your Ceph configuration file, as the default is NOT ideal.
-For details on placement group numbers refer to `setting the number of placement groups`_
+Before creating pools, refer to the `Pool, PG and CRUSH Config Reference`_. If
+you are not using autoscaling you should override the default value for the
+number of placement groups in your Ceph configuration file, as the default is
+NOT ideal.  For details on placement group numbers refer to `setting the number
```

Comment:
Suggest

>If you are not using the PG autoscaler you , may wish to explicitly set a value for :confval:`osd_pool_default_pg_num`, as the default is small and not ideal for many production-scale deployments.  Be careful, though, to not set a very high value as auto-deployed pools, notably certain RGW pools, will not hold much data and thus should not have a gratuitous number of PGs.  When the PG autoscaler is not actively managing placement group numbers, best practice is to explicitly provide `pg_num` and `pgp_num` when creating each pool.

One thing to consider here is that Ceph can deploy pools that don't contain much data and for which a `pg_num` in the hundreds or thousands may be drastic overkill, notably the panoply of pools that RGW uses.

Reply:
Indeed.  I would very much be in favor of merging some of these small pools together.  There's no reason to maintain so many separate pools for RGW.  In the mean time though, it's important to have good behavior in the data and bucket index pools primarily.  The rest are less important (imho).